cmp/compare_test.go

```diff
@@ -1289,448 +1342,504 @@ func embeddedTests() []test {
 		x:     createStructF(0),
 		y:     createStructF(0),
 		opts: []cmp.Option{
 			cmp.AllowUnexported(ts.ParentStructF{}),
 		},
 		wantPanic: "cannot handle unexported field",
+		reason:    "PublicStruct and privateStruct also has unexported fields",
 	}, {
-		label: label + "ParentStructF",
+		label: label + "/ParentStructF/PanicUnexported3",
 		x:     createStructF(0),
 		y:     createStructF(0),
 		opts: []cmp.Option{
 			cmp.AllowUnexported(ts.ParentStructF{}, ts.PublicStruct{}),
 		},
 		wantPanic: "cannot handle unexported field",
+		reason:    "privateStruct also has unexported fields",
 	}, {
-		label: label + "ParentStructF",
+		label: label + "/ParentStructF/Equal",
 		x:     createStructF(0),
 		y:     createStructF(0),
 		opts: []cmp.Option{
 			cmp.AllowUnexported(ts.ParentStructF{}, ts.PublicStruct{}, privateStruct),
 		},
 		wantEqual: true,
+		reason:    "unexported fields of both ParentStructF, PublicStruct, and privateStruct are allowed",
 	}, {
-		label: label + "ParentStructF",
+		label: label + "/ParentStructF/Inequal",
 		x:     createStructF(0),
 		y:     createStructF(1),
 		opts: []cmp.Option{
 			cmp.AllowUnexported(ts.ParentStructF{}, ts.PublicStruct{}, privateStruct),
 		},
 		wantEqual: false,
+		reason:    "the two values differ on some fields",
 	}, {
-		label:     label + "ParentStructG",
+		label:     label + "/ParentStructG/PanicUnexported1",
 		x:         ts.ParentStructG{},
 		y:         ts.ParentStructG{},
 		wantPanic: wantPanicNotGo110("cannot handle unexported field"),
 		wantEqual: !flags.AtLeastGo110,
+		reason:    "ParentStructG has unexported fields",
 	}, {
-		label: label + "ParentStructG",
+		label: label + "/ParentStructG/Ignored",
 		x:     ts.ParentStructG{},
 		y:     ts.ParentStructG{},
 		opts: []cmp.Option{
 			cmpopts.IgnoreUnexported(ts.ParentStructG{}),
 		},
 		wantEqual: true,
+		reason:    "unexported fields of ParentStructG are ignored",
 	}, {
-		label: label + "ParentStructG",
+		label: label + "/ParentStructG/PanicUnexported2",
 		x:     createStructG(0),
 		y:     createStructG(0),
 		opts: []cmp.Option{
 			cmp.AllowUnexported(ts.ParentStructG{}),
 		},
 		wantPanic: "cannot handle unexported field",
+		reason:    "privateStruct also has unexported fields",
 	}, {
-		label: label + "ParentStructG",
+		label: label + "/ParentStructG/Equal",
 		x:     createStructG(0),
 		y:     createStructG(0),
 		opts: []cmp.Option{
 			cmp.AllowUnexported(ts.ParentStructG{}, privateStruct),
 		},
 		wantEqual: true,
+		reason:    "unexported fields of both ParentStructG and privateStruct are allowed",
 	}, {
-		label: label + "ParentStructG",
+		label: label + "/ParentStructG/Inequal",
 		x:     createStructG(0),
 		y:     createStructG(1),
 		opts: []cmp.Option{
 			cmp.AllowUnexported(ts.ParentStructG{}, privateStruct),
 		},
 		wantEqual: false,
+		reason:    "the two values differ on some fields",
 	}, {
-		label:     label + "ParentStructH",
+		label:     label + "ParentStructH/EqualNil",
```

Comment:
Does the string also require / in front?  Same for the next 5 cases below?

I didn't see this in the diff file, so I wasn't sure.

Reply:
Nice catch, yea.

You can see it missing a "/" here:
```
>> TestDiff/EmbeddedStruct/ParentStructG/Inequal
<<< TestDiff/EmbeddedStructParentStructH/Inequal
```